README.md

```diff
@@ -1,2 +1,125 @@
-# Morse_Code_Ruby
-The repository to decode morse code has never been made easier!
+
+<a name="readme-top"></a>
+<div align="center">
+<img src="https://cdn.icon-icons.com/icons2/2415/PNG/512/postgresql_original_wordmark_logo_icon_146392.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Ruby Morse Code</b></h3>
+
+</div>
+
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Vet clinic database ](#-vet-clinic-database-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Create database](#create-database)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+
+# 📖Ruby Morse Code <a name="about-project"></a>
+
+
+**Ruby Morse Code** is a is an algorithm that converts a morse code string to readable English.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+ - Ruby
+
+### Key Features <a name="key-features"></a>
+
+- **Convert Morse Code to English**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- [Ruby Runetime](https://www.postgresql.org/download/) installed
+
+
+### Setup
+
+Clone this repository to your desired folder:
+```
+git clone https://github.com/calebchris000/Morse_Code_Ruby.git && cd Morse_Code_Ruby 
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Mutai Gilbert**
+
+- GitHub: [@githubhandle](https://github.com/Mutai-Gilbert)
+
+👤 **Caleb Nwaizu**
+
+- GitHub: [@githubhandle](https://github.com/calebchris000)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+-  **Optimize Morse code**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Mutai-Gilbert/Vet-Clinic-Database/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+
+If you like this project please give it a ⭐ 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
+
+I would like to thank the Microverse Team! 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE.md) licensed.
```

Comment:
Great job team 🤝

- [ ] [Optional] Kindly fix the broken MIT license link to make your readme file more professional.